### PR TITLE
8904: View Completed version of Order Summary

### PR DESCRIPTION
**Pull request by talawson05**
[AB#9090](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9090)
Test ignored, waiting for 9092 to be complete

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Steps/Steps/CommonSteps.cs [hunk 1]
```diff
@@ -171,7 +171,9 @@ public void GivenACompleteOrderExists()
             order.SupplierName = "Really Kool Corporation";
 
             order.CommencementDate = new Faker().Date.Future().Date;
-            order.DateCompleted = order.CommencementDate.Value.AddDays(1);
+            var dateCompleted = new Faker().Date.Past().Date;
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Steps/Steps/CommonSteps.cs:174]: Faker should be instantiated once and reused

File: src/OrderFormAcceptanceTests.Steps/Steps/CompleteOrder.cs [hunk 2]
```diff
@@ -109,5 +115,41 @@ public void GivenThatTheUserHasCompletedTheirOrder()
             WhenTheUserConfirmsToCompleteTheOrder();
             ThenTheOrderCompletedScreenIsDisplayed();
         }
+
+        [Given(@"a User has completed an Order")]
+        public void GivenAUserHasCompletedAnOrder()
+        {
+            new CommonSteps(Test, Context).GivenACompleteOrderExists();
+        }
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Steps/Steps/CompleteOrder.cs:123]: This Given should be added to the step it references

File: src/OrderFormAcceptanceTests.Steps/Steps/CommonSteps.cs [hunk 3]
```diff
@@ -297,4 +302,4 @@ public void AssertListOfStringsIsInAscendingOrder(IEnumerable<string> stringList
             hexList.Should().BeInAscendingOrder();
         }
     }
-}
+}
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.Steps/Steps/CommonSteps.cs:305]: Should end with a newline to keep git happy.

File: src/OrderFormAcceptanceTests.Tests/Features/CompleteOrder.feature [hunk 4]
```diff
@@ -143,4 +143,11 @@ Scenario: Complete Order - Indicate if automatically processed or not if Funding
...
-	Then there is an indication that the Order has not been processed automatically
\ No newline at end of file
+	Then there is an indication that the Order has not been processed automatically
+@ignore
+Scenario: View Completed Order Summary
+	Given a User has completed an Order 
+	When they choose to view the Completed Order from their Organisation's Orders Dashboard
+	Then the Completed version of the Order Summary is presented
+	And the completed order summary has specific content related to the order being completed
+	And the completed order summary contains the date the Order was completed
```
Comment (bjss-chris) [inline src/OrderFormAcceptanceTests.Tests/Features/CompleteOrder.feature:153]: Should end with a newline to keep git happy.

**Pull request merged**